src/utils/hammer.browser.js

```diff
@@ -1,9 +1,32 @@
-import hammerjs from 'hammerjs';
+// Copyright (c) 2017 Uber Technologies, Inc.
+//
+// Permission is hereby granted, free of charge, to any person obtaining a copy
+// of this software and associated documentation files (the "Software"), to deal
+// in the Software without restriction, including without limitation the rights
+// to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+// copies of the Software, and to permit persons to whom the Software is
+// furnished to do so, subject to the following conditions:
+//
+// The above copyright notice and this permission notice shall be included in
+// all copies or substantial portions of the Software.
+//
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+// IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+// AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+// LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+// OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+// THE SOFTWARE.
+
 import {enhancePointerEventInput, enhanceMouseInput} from './hammer-overrides';
 
-enhancePointerEventInput(hammerjs.PointerEventInput);
-enhanceMouseInput(hammerjs.MouseInput);
+let hammerjs = null;
 
-export const Manager = hammerjs.Manager;
+if (typeof document !== 'undefined') {
+  // Avoid crash if imported in a web worker
+  hammerjs = require('hammerjs');
```

Comment:
Hey @Pessimistress. I just upgraded and it broke my build again. Please NEVER mix commonjs and esm. It's webpackism which is not handled everywhere else well.
My change was intended to fix this. And you reverted it back.
https://github.com/uber-web/mjolnir.js/pull/22

Reply:
Why this module is run inside of worker?

Reply:
🤦🏻‍♀️sorry about that. This was thrown in a while ago when we experimented with running deck.gl on a webworker.

I will introduce eslint-plugin-import to block dynamic require.

Reply:
So you can just revert to my previous solution, right?